contracts/common/misc/Drainable.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.5.2;
+
+import { Governable } from "../governance/Governable.sol";
+
+import {IERC20} from "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import {IERC721} from "openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
+
+
+contract Drainable is Governable {
```

Comment:
Inherit from `DepositManagerStorage` instead. Proxy upgradeable contracts need to have same memory layout.